pkg/eventlog/multireader.go

```diff
@@ -120,5 +120,6 @@ func exists(event Event, log []Event, checker storage.Checker) bool {
 		}
 	}
 
-	return checker.Exists(context.TODO(), event.Module, event.Version)
+	exists, _ := checker.Exists(context.TODO(), event.Module, event.Version)
```

Comment:
im not familiar with eventlog so im not sure why exists gets called here. But we shouldn't ignore this error. CC @arschles can you please check this.

Reply:
event log can go away, olympus will sync using logical pointer, proxy downloads only what was requested